python/cudf/cudf/_libxx/merge.pyx

```diff
@@ -0,0 +1,75 @@
+from cudf._libxx.lib cimport *
+from cudf._libxx.column cimport Column
+from cudf._libxx.table cimport Table
+from libcpp.vector cimport vector
+cimport cudf._libxx.includes.merge as cpp_merge
+
+
+def merge_sorted(
+    tables, keys=None, index=False, ascending=True, na_position="last"
+):
+    cdef vector[size_type] c_column_keys
+    cdef vector[table_view] c_input_tables
+    cdef vector[order] c_column_order
+    cdef vector[null_order] c_null_precedence
+    cdef order column_order
+    cdef null_order null_precedence
+    cdef Table source_table
+
+    # Create vector of tables
+    # Use metadata from 0th table for names, etc
+    for source_table in tables:
+        c_input_tables.push_back(source_table.view())
```

Comment:
We should reserve the size up front so we don't need to constantly resize here

Reply:
@kkraus14 -  What is the correct way to initialize the size of a `vector[size_type]`  without specifying a fill value?  I expected it to be something like `vector[size_type](num_key_columns)`, but that seems to fail the "cythonize" step (Sorry for my cython ignorance)

Reply:
You should be able to just do `vector[table_view](size_type)` the `[...]` is equivalent to C++ `<...>` where it defines the type inside the vector.

Reply:
Sorry - I made this comment/question in the wrong place.  It seems to work fine for `vector[table_view]`, but **not** for vector[size_type] (I'm sure I'm doing something silly, but I can't tell what that is yet)

Reply:
Are you sure you've defined `size_type` as a `ctypedef`? I.E. you'd need to `cimport` this: https://github.com/rapidsai/cudf/blob/branch-0.13/python/cudf/cudf/_libxx/lib.pxd#L17